core/modules/system/system.install

```diff
@@ -531,59 +531,45 @@ function system_requirements($phase) {
-    $path = backdrop_get_path('module', 'redirect');
-    if ($path && strpos($path, 'core/modules/redirect') === FALSE) {
-      $requirements['redirect_replaced'] = array(
-        'title' => $t('Redirect'),
-        'value' => $t('Duplicate module detected'),
-        'severity' => REQUIREMENT_WARNING,
-        'description' => $t('Backdrop core now provides a bundled Redirect module. A different copy of Redirect module has been located at %path. Remove this module from your installation to use the new Redirect module. For more information see the <a href="https://api.backdropcms.org/node/43994" target="_blank">Redirect change notice</a>.', array('%path' => BACKDROP_ROOT . '/' . $path)),
-      );
+        // For certain core modules, we add some additional explanation.
+        $path = substr($file->uri, 0, strlen($file->uri) - strlen($file->name) - 8);
+        if ($file->name == 'ckeditor') {
+          $description .=  $t('Backdrop core provides a bundled CKEditor module. A different copy of CKEditor module is at %path. To use the new CKEditor module, disable and then uninstall the current module, remove it from your installation, and then enable the core CKEditor module from the modules page. For more information see the <a href="https://api.backdropcms.org/node/42827" target="_blank">CKEditor change notice</a>.', array('%path' => $path));
+        }
+        elseif ($file->name == 'email') {
+          $description .= $t('Backdrop core provides a bundled Email module. A different copy of Email module is at %path. Remove this module from your installation to use the core Email module. For more information see the <a href="https://api.backdropcms.org/node/42928" target="_blank">Email module change notice</a>.', array('%path' => $path));
+        }
+        elseif ($file->name == 'link') {
+          $description .= $t('Backdrop core provides a bundled Link module. A different copy of Link module is at %path. Remove this module from your installation to use the core Link module. For more information see the <a href="https://api.backdropcms.org/node/42930" target="_blank">Link change notice</a>.', array('%path' =>  $path));
+        }
+        elseif ($file->name == 'date') {
+          $description .= $t('Backdrop core provides a bundled Date module. A different copy of Date module is at %path. Remove this module from your installation to use the core Date module. For more information see the <a href="https://api.backdropcms.org/node/43066" target="_blank">Date change notice</a>.', array('%path' => $path));
+        }
+        elseif ($file->name == 'redirect') {
+          $description .= $t('Backdrop core provides a bundled Redirect module. A different copy of Redirect module is at %path. Remove this module from your installation to use the core Redirect module. For more information see the <a href="https://api.backdropcms.org/node/43994" target="_blank">Redirect change notice</a>.', array('%path' => $path));
+        }
```

Comment:
This could be consolidated into a single string and used for all of these (and future ones):

```
           $description .= $t('Backdrop core provides a bundled @name module. A different copy of @name module is at %path. Remove this module from your installation to use the core @name module. For more information see the <a href="!url" target="_blank">@name change notice</a>.', array('%path' => $path, '@name' => t('Redirect'), '!url' => 'https://api.backdropcms.org/node/43994'));
```

Reply:
If we make that change, then all of the modules will be sent to the change record for Redirect. Note that the node ID is  different in each URL. They can still be consolidated, but we'll need a switch statement for the URL. (Also we can take the opportunity to use the docs.b.org URL, rather than api.b.org.)